tests/ci_build/tidy.py

```diff
@@ -234,11 +239,13 @@ def test_tidy():
     parser = argparse.ArgumentParser(description='Run clang-tidy.')
     parser.add_argument('--cpp', type=int, default=1)
     parser.add_argument('--cuda', type=int, default=1)
+    parser.add_argument('--bundle', type=int, default=1,
```

Comment:
Not sure how useful this argument is going to be, since currently, `tidy.py` doesn't let the user specify the path of google test. Either we should add `--gtest-path` option or remove `--bundle` option. My personal preference is to always use the bundled gtest.

Reply:
I installed gtest on my system path so CMake always handles it for me...

Reply:
In this case, let's rename the option to `--use-system-gtest`